_projects/2024/gnuastro/opencl_integration.md

```diff
@@ -0,0 +1,67 @@
+---
+name: OpenCL integration with Gnuastro
+desc: GPU support inside Gnuastro C library.
+requirements:
+# Student requirements:
+ -C, OpenCL, GNU Autotools (build system)
```

Comment:
requirements don't render, preferably you would do a half sentence description of the expected level of familiarity, too (see other project ideas)
```suggestion
 - C
 - OpenCL
 - GNU Autotools (build system)
```